simplekv/__init__.py

```diff
@@ -418,3 +435,42 @@ class UrlKeyValueStore(UrlMixin, KeyValueStore):
        Use the :class:`.UrlMixin` instead.
     """
     pass
+
+
+class CopyMoveMixin(object):
+    """Exposes a copy and move API. This API is either backed by corresponding backend operations, if available,
+    or emulated using get/put/delete.
+
+    Warning: This makes the operations potentially not atomic!"""
```

Comment:
Maybe a bit softer: `Note: Copy and move operations are not guaranteed to be atomic.`